english/watched/watched.md

```diff
@@ -1,4 +1,3 @@
 You gaze into the abyss, and the abyss gazes into you.
 
-Perhaps you'd like to lighten up and [bake some cookies.](../bake-cookies/cookies.md)
-
+Perhaps you'd like to lighten up and [bake some cookies.](../bake-cookies/cookies.md) Or, you may decide it's easier just to go to the bakery and [buy them.](buy-cookies/buy-cookies.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.